### PR TITLE
prefer specifying slice capacity for appending

### PR DESCRIPTION
**Pull request by hothero**

### CODE REVIEW

File: style.md [hunk 1]
```diff
@@ -1182,6 +1183,51 @@ allocations at assignment time.
 </td></tr>
 </tbody></table>
 
+### Prefer Specifying Slice Capacity for Appending
+
+Where possible, provide capacity value when initialize a slice for appending with `make()`.
```
Comment (mlaventure) [inline style.md]:
```suggestion
Where possible, provide capacity value when initializing a slice for appending with `make()`.
```

**Comment by mway** [comment]
Hey @hothero! Friendly ping - did you want to take a stab at the feedback? If so, great! If not, no problem - just let us know.

Thanks!

**Comment by hothero** [comment]
@mway thanks for the kindly reminder :) I think that's a good idea. I'd prefer to do that in different PR. So I'm not going to make the change in this PR.

File: style.md [hunk 2]
```diff
@@ -1182,6 +1183,51 @@ allocations at assignment time.
 </td></tr>
 </tbody></table>
 
+### Prefer Specifying Slice Capacity for Appending
+
+Where possible, provide capacity value when initializing a slice for appending with `make()`.
```
Comment (mway) [inline style.md:1188]:
```suggestion
Where possible, provide a capacity value to `make()` when initializing a slice for appending.
```

**Comment by mway** [comment]
Thanks @hothero - mind signing the CLA, please?

**Comment by hothero** [comment]
@mway signed. Thanks!

**Pull request merged**